src/Libraries/ReactNative/I18nManager.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+'use strict';
```

Comment:
This line causes a linter error: `10:1  error  'use strict' is unnecessary inside of modules  strict`  

Do you mind updating this pull request to make sure all of the tests and linter pass?

Reply:
Hey! Thanks for taking a look. Unfortunately I was hacking on this for a currently-shelved hobby project and so don't have my build/etc. all still around. It'd be great if you could fix the linter error while merging/testing the PR. (If there are tests failing, agreed that is a bigger deal/not something I'd ask you to go mucking around in.)